development/roles.md

```diff
@@ -257,22 +263,32 @@ technologies.
 
 The following subproject repositories and subproject maintainers are configured:
 
-* [charts]({{site.github_org_url}}/charts)
-  * Jan Waś [<i class="fab fa-github"></i> nineinchnick](https://github.com/nineinchnick)
-* [grafana-trino]({{site.github_org_url}}/grafana-trino)
-  * Jan Waś [<i class="fab fa-github"></i> nineinchnick](https://github.com/nineinchnick)
-* [trino.io]({{site.github_org_url}}/trino.io)
-  * Brian Olsen [<i class="fab fa-github"></i> bitsondatadev](https://github.com/bitsondatadev)
-  * Cole Bowden [<i class="fab fa-github"></i> colebow](https://github.com/colebow)
-* [trino-gateway]({{site.github_org_url}}/trino-gateway)
-  * Jaeho Yoo [<i class="fab fa-github"></i> Chaho12](https://github.com/Chaho12)
-  * Star Poon [<i class="fab fa-github"></i> oneonestar](https://github.com/oneonestar)
-  * Vishal Jadhav [<i class="fab fa-github"></i> vishalya](https://github.com/vishalya)
-  * Will Morrison [<i class="fab fa-github"></i> willmostly](https://github.com/willmostly)
-* [trino-go-client]({{site.github_org_url}}/trino-go-client)
-  * Jan Waś [<i class="fab fa-github"></i> nineinchnick](https://github.com/nineinchnick)
-* [trino-js-client]({{site.github_org_url}}/trino-js-client)
-  * Filipe Regadas [<i class="fab fa-github"></i> regadas](https://github.com/regadas)
+[aws-proxy]({{site.github_org_url}}/aws-proxy) - Proxy for AWS S3
```

Comment:
Can we link to the team on github? like this one is https://github.com/orgs/trinodb/teams/maintainers-s3-proxy.  I'm not sure if that is publically visible though.

Reply:
requires a github account AND maintainer ship I think .. at least I got others to test and they see a 404 ..